includes/Integrations/Jetpack.php

```diff
@@ -128,11 +128,18 @@ public function filter_ajax_query_attachments_args( array $args ) {
 			return $args;
 		}
 
-		$allowed_mime_types = $this->get_allowed_mime_types();
-		$allowed_mime_types = array_merge( ...array_values( $allowed_mime_types ) );
-		if ( in_array( self::VIDEOPRESS_MIME_TYPE, $args['post_mime_type'], true ) && ! array_diff( $allowed_mime_types, $args['post_mime_type'] ) ) {
-			// Load filter at 15, so it load after Media\Media\wp_prepare_attachment_for_js which is loaded at 10.
-			add_filter( 'wp_prepare_attachment_for_js', [ $this, 'filter_admin_ajax_response' ], 15, 2 );
+		$allowed_mime_types              = $this->get_allowed_mime_types();
+		$allowed_mime_types              = array_merge( ...array_values( $allowed_mime_types ) );
+		$allowed_mime_types_transcodable = array_merge( $allowed_mime_types, $this->get_allow_transcodable_mime_types() );
```

Comment:
This appears to be the only reason why `get_allow_transcodable_mime_types` is needed in PHP, is that correct? And it's only needed here to determine whether the ajax request is coming from our editor?

Reply:
Yeah, I did not want to maintain two lists here. I also think this is a example of a where this value should be populated from PHP, so it maybe needed in other context in the future in PHP.